lib/core/utils.js

```diff
@@ -163,6 +163,10 @@ function isUnifiedTopology(topology) {
   return topology.description != null;
 }
 
+function arrayStrictEqual(arr, arr2) {
```

Comment:
Are we ever going to have a case where the arrays we are using this on do not exist? Should we add an `Array.isArray(arr)`?

Reply:
added